OBAKit/Stops/StopViewController.swift

```diff
@@ -691,7 +690,17 @@ public class StopViewController: UIViewController,
     }
 
     public func listAdapter(_ listAdapter: ListAdapter, sectionControllerFor object: Any) -> ListSectionController {
-        return defaultSectionController(for: object)
+        let controller = defaultSectionController(for: object)
+
+        // Set delegates
+        switch controller {
```

Comment:
Is there a reason you didn't express this as:

```
if let serviceAlertsController as? ServiceAlertsSectionController {
    serviceAlertsController.delegate = self
}
```

The switch statement feels like it obscures your intent.

Reply:
i kinda forgot, but i think the intention was to future proof it in case we have more delegates to set in the future.